modules/n1ql/examples/n1ql-language-reference/udf-locations-define.n1ql

```diff
@@ -0,0 +1,4 @@
+CREATE FUNCTION locations(vActivity) { (
+  SELECT id, name, address, city
+  FROM `travel-sample`.inventory.landmark
+  WHERE activity = vActivity) };
```

Comment:
If this is a *new* query, is it worth adding ordering from the get-go to avoid the data-change artefacts we discussed recently?

Reply:
Ah I see, it's not new, just re-homed. As you were...